include/tvm/node/reflection.h

```diff
@@ -43,7 +43,7 @@ using runtime::ObjectPtr;
 using runtime::ObjectRef;
 
 /*!
- * \brief Visitor class for to get the attributesof a AST/IR node.
+ * \brief Visitor class to get the attributes of a AST/IR node.
```

Comment:
```suggestion
 * \brief Visitor class to get the attributes of an AST/IR node.
```

Reply:
aargh @tkonolige Thanks! Done in https://github.com/apache/tvm/pull/8787/commits/d98f7b81951c93e4444c3413cf6889388d98e4ef

Reply:
Thanks!